airflow/bin/cli.py

```diff
@@ -354,7 +374,8 @@ def run(args, dag=None):
         # via the UI (or vice versa) results in a permission error as the task
         # tries to write to a log file created by the other user.
         log_base = os.path.expanduser(conf.get('core', 'BASE_LOG_FOLDER'))
-        directory = log_base + "/{args.dag_id}/{args.task_id}".format(args=args)
+        log_relative_dir = logging_utils.get_log_directory(**locals())
```

Comment:
@bolkedebruin Good note I've changed the `**local()` to be something more specific. I guess in the future I shouldn't follow whatever the codebase has already since it uses `**local()` very often.